source/PlayerInfo.cpp

```diff
@@ -157,7 +157,24 @@ void PlayerInfo::Load(const string &path)
 		else if(child.Token(0) == "travel" && child.Size() >= 2)
 			travelPlan.push_back(GameData::Systems().Get(child.Token(1)));
 		else if(child.Token(0) == "travel destination" && child.Size() >= 2)
+		{
```

Comment:
This statement is duplicated. I'd rather put refactor this into a function, but I don't know where to put such a function.